test/e2e/common.ts

```diff
@@ -207,3 +227,6 @@ export async function makeCoordinatorZomeBundle(): Promise<CoordinatorBundle> {
     },
   };
 }
+
+export const delay = (ms: number) =>
```

Comment:
Do I see it correctly that this is not actually used anywhere?

Reply:
yep good catch, removed: https://github.com/holochain/holochain-client-js/pull/336/commits/4bcaced7dd74c843b1d1d41c5157f5444bd68735